public/docs/ts/latest/guide/animations.jade

```diff
@@ -0,0 +1,365 @@
+include ../_util-fns
+
+:marked
+  Motion is an important aspect in the design of modern web applications. We want our
+  user interfaces to have smooth transitions between states, and engaging animations
+  that call attention where it's needed. Well-designed animations can make a UI not only
+  more fun but also easier to use.
+
+  Angular's animation system gives us what we need to make the kinds of animations we want.
+  We can build animations that run with the same kind of native performance that we're used
+  to with pure CSS animations. But we can also have our animation logic tightly integrated
+  with the rest of our application code, where they can be easily triggered and controlled.
+
+.alert.is-helpful
+  :marked
+    Angular animations are built on top of the standard [Web Animations API](https://w3c.github.io/web-animations/)
+    and they run natively on [browsers that support it](http://caniuse.com/#feat=web-animation).
+
+    For other browsers, a polyfill is required. Grab
+    [`web-animations.min.js` from here](https://github.com/web-animations/web-animations-js) and
+    add it to your page.
+
+    A more lightweight polyfill maintained by the Angular team is coming soon.
+
+:marked
+  # Table of Contents
```

Comment:
Something like this would be great as part of the the Hero HTML; perhaps using `mouseOver` effects.